README.md

```diff
@@ -103,11 +86,26 @@ $ git push <your_remote> <your_new_branch>
 9. Repeat this process until we will have done amazing freezer
 
 
+### Used tools
+
+* [PEP 3333 - Python WSGI](https://www.python.org/dev/peps/pep-3333/)
+* [flask](https://flask.palletsprojects.com/en/1.1.x/)
+* [pytest](https://docs.pytest.org/en/latest/)
+
+
+### Why start this project
+
+The Czech Python community uses a lot of static web pages that are generated from a web application for community purposes e.g. workshops, courses, or meetups.
+
+The community has been so far relying on the following projects ([flask-frozen](https://pythonhosted.org/Frozen-Flask/) and [elsa](https://github.com/pyvec/elsa/)) in order to generate the static web content. 
+
+
+The new freezer ought to be used with any arbitrary Python Web application framework ([Flask](https://flask.palletsprojects.com/en/1.1.x/), [Django](https://www.djangoproject.com/), [Tornado](https://www.tornadoweb.org/en/stable/), etc.) and with our new [freezer](https://github.com/encukou/freezeyt).
```

Comment:
>... and with our new [freezer](https://github.com/encukou/freezeyt).

Tahle část mi přijde celkem matoucí, jakoby ses snažil říct, že ten freezer bude kompatibilní sám se sebou.

Reply:
jo to je pravda zkusím to přepsat asi úplně celé až budu mít čas